agent/app/agent_capability_test.go

```diff
@@ -815,6 +815,12 @@ func TestCapabilitiesExecuteCommand(t *testing.T) {
 			getSubDirectories:        func(path string) ([]string, error) { return []string{"3.0.236.0"}, nil },
 			shouldHaveExecCapability: true,
 		},
+		{
+			name:                     "execute-command capability should be added if have valid ssm version exists",
+			pathExists:               func(path string, shouldBeDirectory bool) (bool, error) { return true, nil },
```

Comment:
could you also add a test case where dir exists, but have invalid binaries and exec capability is false?

Reply:
yeah, sure, will add it!